### ISSUE TITLE
Make app navbar responsive for mobile

### ISSUE THREAD

**Issue by growupanand** [growupanand/crm-webapp#148]
Currently navbar have no mobile version, so we have to make navbar responsive for mobile devices.

current navbar screenshot
<img width="459" alt="image" src="https://github.com/growupanand/crm-webapp/assets/29487686/218548a9-e358-4ca0-b38c-f9a2c99db6e7">

**Comment by ravi0900** [comment]
@growupanand I want to contribute, can you please assign this to me

**Comment by ravi0900** [comment]
@growupanand also I'm unable to create account in vercel link, is everything working alright in backend.

**Comment by SAteroid** [comment]
please assign this issue to me

**Comment by growupanand** [comment]
> please assign this issue to me

@SAteroid I've already assigned this task to @ravi0900. However, there are still more opportunities for UI improvements. If you're interested, please let me know, and I can either create additional tasks for you or you can work on them yourself.

**Comment by SAteroid** [comment]
@growupanand  Yes I am interested in contributing , please create a issue for me with hacktoberfest label.

**Comment by growupanand** [comment]
> @growupanand Yes I am interested in contributing , please create a issue for me with hacktoberfest label.

@SAteroid you can pick this issue - [Create a Dark Theme Color Scheme](https://github.com/growupanand/crm-webapp/issues/150)

**Comment by ravi0900** [comment]
> Hey @ravi0900 , I just wanted to make sure you were still on it.

can you please guide me to setup env file to run the project. I give up after seeing env file, But I can do it now if you help me, ready to join zoom/meet as well

**Comment by ravi0900** [comment]
@growupanand added a pr #156 
please review what is failing, everything was working fine locally.
